cclient/request/broadcast_transactions.c

```diff
@@ -0,0 +1,17 @@
+#include "broadcast_transactions.h"
```

Comment:
please add the full standard copyright. see `add_neighbors.c`

Reply:
Okay.

Reply:
Copyright still missing